scripts/build-info.py

```diff
@@ -0,0 +1,53 @@
+# Copyright Concurrent Technologies Corporation 2023
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import json
+import subprocess
```

Comment:
I said earlier that subprocess shouldnt be in the PR, that was incorrect. Your usage here is fine.  I should have said it shouldnt be used at runtime, only at build time.

:+1: